examples/visualizer/main.py

```diff
@@ -0,0 +1,292 @@
+import argparse
+import cProfile
+import pstats
+from os import getenv
+from pstats import SortKey
+
+import psycopg2
+import rustworkx as rx
+from rustworkx.visualization import graphviz_draw
+from tabulate import tabulate
+
+CHAI_DATABASE_URL = getenv("CHAI_DATABASE_URL")
```

Comment:
this should be in the README, right? maybe with a fall-back to our default docker config?